package.json

```diff
@@ -1,25 +1,22 @@
 {
-  "name": "@nui/node-cloudstorage",
-  "version": "5.0.0",
+  "name": "@adobe/cloudstorage",
```

Comment:
I think we should use a more specific name here. This does sound pretty broad, or one might assume it's some form of Adobe platform storage.

It is focused on
- _blob_ storage
- thin wrapper for presigned urls, upload, put & listing

Maybe `@adobe/cloud-blob-storage`?

cc @DrGirlfriend @jdelbick @bobvanmanen

Reply:
I changed the name to `@adobe/cloud-blob-storage`.

@alexkli Can you rename the repo so the names fit?

Reply:
@alexkli How do I use credentials in the tests? Are they in a file? Do I access them through an env variable?

Current error: `INFO: AZURE_STORAGE_KEY and/or AZURE_STORAGE_ACCOUNT are not set, trying file: /home/travis/.adobe-asset-compute/credentials.yaml`

Reply:
Let's get feedback from the group. I am not sure `@adobe/cloud-blob-storage` is good either...

Reply:
@pheenomenon @DrGirlfriend @jdelbick ^^^^^

Reply:
Plus Azure & Google call it blob and aws calls it object....
virtual-storage?
cloud-services?
cloud-service-storage?

This is going to be hard

Reply:
`cloud-binary-storage` ?